opentok/opentok.py

```diff
@@ -91,6 +93,22 @@ def proxies(self):
     def proxies(self, proxies):
         self._proxies = proxies
 
+    @property
```

Comment:
Not really certain what these properties are meant to be doing in here?

Reply:
This is part of a set of major breaking changes and I will tag this as draft because we haven't agreed on the convention yet.